o8g/scripts/actions.py

```diff
@@ -337,6 +337,7 @@ def flip(card, x=0, y=0):
     card.alternate = ALTERNATE_POLITICAL if not card.alternate else ''
     return
   card.isFaceUp = not card.isFaceUp
+  notify("{} flips {}.".format(me,card))
```

Comment:
Would also be good to add a mute() to just the if section in case octgn adds messages for those commands